src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -42,25 +43,45 @@ internal class MessageReceiverExample : MessageReceiver {
     }
 }
 
+class SubscribeManager(subscriptionId: String) {
+    private val dockerProcessor = DockerProcessor()
+    private val subscriptionName = SubscriptionName.of(PROJECT_ID, subscriptionId)
+    private var subscriber: Subscriber? = null
```

Comment:
You use this in a single method only, so you can make it a local val in that method.

Reply:
Done